pkg/redisqueue/consumer.go

```diff
@@ -168,6 +169,21 @@ func (c *Consumer) dequeue(ctx context.Context) (*redisqueue.Task, *deps.AppProv
 	return &task, appProvider, err
 }
 
+func (c *Consumer) process(
+	ctx context.Context,
+	task *redisqueue.Task,
+	appProvider *deps.AppProvider,
+) (result json.RawMessage, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = fmt.Errorf("task panic: %s", r)
```

Comment:
Use panicutil.MakeError to turn the return value of recover() into an error.

Reply:
Fixed.